worker/games.py

```diff
@@ -250,6 +253,8 @@ def run_game(p, remote, result, spsa, spsa_tuning, tc_limit):
         spsa['wins'] = wld[0]
         spsa['losses'] = wld[1]
         spsa['draws'] = wld[2]
+        if spsa['num_games'] == spsa['wins'] + spsa['losses'] + spsa['draws']:
```

Comment:
I think it would be good to add a comment here, at least I found it difficult to understand why this works until I noticed that the condition in the server code https://github.com/glinscott/fishtest/blob/e93e526ee0c3dbe2dc2524ec4b3131b4a170e79f/fishtest/fishtest/rundb.py#L298-L300
is exactly the same and the parameters are not used otherwise.